lib/builder/syncer.go

```diff
@@ -70,12 +69,8 @@ func newS3Syncer() (*s3Syncer, error) {
 
 // Sync makes sure that local cache has all required dependencies for the
 // selected runtime
-func (s *s3Syncer) Sync(builder *Builder, runtimeVersion *semver.Version) error {
-	tarball, err := s.hub.Get(loc.Locator{
-		Repository: defaults.SystemAccountOrg,
-		Name:       defaults.TelekubePackage,
-		Version:    runtimeVersion.String(),
-	})
+func (s *s3Syncer) Sync(ctx context.Context, builder *Builder, runtimeVersion semver.Version) error {
+	tarball, err := s.hub.Get(loc.Gravity.WithVersion(runtimeVersion))
```

Comment:
Have you made sure that open-source tele build still works with this change? I don't remember if "hub" used here supports "telekube <-> gravity" name translation.

Reply:
Good catch - I've changed the application to `telekube` for the oss version so this works.